apps/admin-x-activitypub/src/components/feed/FeedItem.tsx

```diff
@@ -250,8 +238,8 @@ const FeedItem: React.FC<FeedItemProps> = ({actor, object, layout, type, comment
     // });
 
     const UserMenuTrigger = (
-        <Button
-            className={`transition-color relative z-[9998] flex h-[34px] w-[34px] items-center justify-center ${layout === 'feed' ? 'rounded-full' : 'rounded-md'} bg-white hover:bg-gray-100 dark:bg-black dark:hover:bg-gray-950 ${(layout === 'feed' || layout === 'modal') && 'ml-auto'}`}
+        <ButtonX
```

Comment:
We should replace and use the `Button` component from Shade instead of the old system. What's the reason to keep the old one here?